src/plugins/formulas/formulas.js

```diff
@@ -762,7 +810,12 @@ export class Formulas extends BasePlugin {
    * @returns {*|boolean} If false is returned the action is canceled.
    */
   onBeforeCreateRow(row, amount) {
-    if (!this.engine.isItPossibleToAddRows(this.sheetId, [this.toPhysicalRowPosition(row), amount])) {
+    // Using `attempt` because this.sheetId can be null or point to an non-existing sheet
```

Comment:
So we should check it correctly. Not suppress the error
```suggestion
    if (this.sheetId && this.engine.doesSheetExist(this.sheetId)) {
```

Reply:
I extended these conditions in https://github.com/handsontable/handsontable/pull/8669/commits/c52a4f85f0966f7c2eeed8e267a374aff3480695.